src/components/util/search-matching-equipments.js

```diff
@@ -26,40 +27,42 @@ export const useSearchMatchingEquipments = (
     const [useNameLocal] = useParameterState(PARAM_USE_NAME);
 
     const searchMatchingEquipments = useCallback(
-        (searchTerm) => {
+        (searchTerm, sooner = false) => {
```

Comment:
"sooner" variable name is unclear, change it to give some clue about it's behaviour

Reply:
What is unclear about adjective 'sooner', especially given a default value of 'false' ?
https://www.google.com/search?channel=fs&q=traduire+sooner

Reply:
my point here is the purpose of the variable, what does "sooner" means for "searchMatchingEquipment" ? It should search sooner ? Sooner than what ?

Reply:
sooner than later -> do not wait much before really asking